pegasus/router.rb

```diff
@@ -145,6 +145,13 @@ def self.set_max_age(type, default)
     redirect "/#{path}"
   end
 
+  # Page mode selection
+  get '/private/pm/*' do |page_mode|
+    dont_cache
+    response.set_cookie('pm', {value: page_mode, domain: ".#{request.site}", path: '/', expires: Time.now + 3600*4})
```

Comment:
this looks like a 4 hour cookie?  there is also a session cookie, in which expires is not set, if we preferred that..